opm/simulators/linalg/bda/opencl/openclKernels.hpp

```diff
@@ -64,6 +64,7 @@ class OpenclKernels
     static cl::CommandQueue *queue;
     static std::vector<double> tmp;     // used as tmp CPU buffer for dot() and norm()
     static bool initialized;
+    static size_t preferred_workgroup_size_multiple; // stores CL_KERNEL_PREFERRED_WORK_GROUP_SIZE_MULTIPLE
```

Comment:
You should include `<stddef.h>` if you have data members of type `size_t`.  Alternatively, you could/should make the data member into a `std::size_t` and include `<cstddef>` instead.

Reply:
18d9f8c, uses `<cstddef>`.